### PR TITLE
Clarify that <fieldset> only uses the disabled attribute

### LINKED ISSUE #16630: Docs falsely claim "not-allowed" cursor for labels inside fieldset.disabled
From http://getbootstrap.com/css/#checkboxes-and-radios:
> To have the <label> for the checkbox or radio also display a "not-allowed" cursor when the user hovers over the label, **add the .disabled class to** your .radio, .radio-inline, .checkbox, .checkbox-inline, or **&lt;fieldset>**

(bold highlighting by me)

Either the doc is wrong or the theme is missing something but adding the disabled class to an encompassing fieldset makes no difference.

Testcase with comparison to the other methods leading to the wanted result: http://jsbin.com/kacawu/3/edit?html,output (Bootlint and HTML validator passed).

### PR DESCRIPTION
**Pull request by kkirsche**
Fixes #16630

Not sure the exact way to word it, but let me know if you'd like adjustments and I'll make those and rebase

### CODE REVIEW

**Comment by zacechola** [comment]
A slight reorganization could work here, by breaking the paragraph into two parts: the attribute and the class, while also maintaining your cross link.

> Only the `disabled` attribute is required on checkboxes, radios and [fieldsets](#forms-disabled-fieldsets) to style the elements appropriately. To have the `<label>` for the checkbox or radio also display a "not-allowed" cursor when the user hovers over the label, add the `.disabled` class to your `.radio`, `.radio-inline`, `.checkbox`, `.checkbox-inline`.

**Comment by patrickhlauke** [comment]
agree with @zacechola

**Comment by kkirsche** [comment]
Cool I'll make that change when I get home per your recommendation @zacechola and rebase it. Thanks for the feedback so we can ensure this is clear :+1:

**Comment by mdo** [comment]
I think all we need here is:

> Disabled checkboxes and radios are supported, but to provide a "not-allowed" cursor on hover, you'll need to add the `.disabled` class to the parent `.radio`, `.radio-inline`, `.checkbox`, or `.checkbox-inline`.

It feels odd to mention `<fieldset>`s here at all. That's for the disabled fieldset docs section, not the checkboxes and radios. Sound about right?

**Comment by kkirsche** [comment]
I can understand that @mdo. The thought being why mention something is disabled if the section isn't about the component. Does that sound accurate for the reasoning behind nuking that component from the wording there? If so, that makes sense to me.

**Comment by kkirsche** [comment]
Pushed your recommendation @mdo — let me know if any other changes or adjustments are needed or desired.

File: docs/_includes/components/input-groups.html [hunk 1]
```diff
@@ -26,6 +26,7 @@ <h2 id="input-groups-basic">Basic example</h2>
   <p>Place one add-on or button on either side of an input. You may also place one on both sides of an input.</p>
   <p><strong class="text-danger">We do not support multiple add-ons on a single side.</strong></p>
   <p><strong class="text-danger">We do not support multiple form-controls in a single input group.</strong></p>
+  <p><strong class="text-danger"><code>input-group-addon</code> must be the last child within an input group.</strong></p>
```
Comment (mdo) [inline docs/_includes/components/input-groups.html]: Why the addition of this?

Comment (kkirsche) [inline docs/_includes/components/input-groups.html]: That's a fantastic question. No clue where this even came from as I haven't opened that file

**Comment by kkirsche** [comment]
removed that weird addition @mdo — not sure where it came from. Sorry about that

**Comment by mdo** [comment]
Guessing there was something wrong with the force push? The commit message is now inaccurate :).

**Comment by kkirsche** [comment]
My guess is I had a leftover patch-* branch locally that this overlapped with

**Comment by mdo** [comment]
Thanks for all the back and forth! Sorry for nit picking :grin:.

**Comment by kkirsche** [comment]
Not nit picking at all. Legitimate mistakes on my part that should have been fixed before even sending it out to you guys. Gonna clean up my branches to only my open PR's so hopefully this won't happen again. Thanks for your patience

**Pull request merged**